packages/client/src/generation/TSClient/jsdoc.ts

```diff
@@ -146,68 +151,73 @@ const ${lowerCase(ctx.mapping.model)} = await ${ctx.method}({
     body: (ctx) => {
       const onlySelect = ctx.firstScalar
         ? `\n// Only select the \`${ctx.firstScalar.name}\`
-const ${lowerCase(ctx.mapping.model)}With${capitalize(ctx.firstScalar.name)}Only = await ${ctx.method}({ select: { ${
+const resultsWith${capitalize(ctx.firstScalar.name)}Only = await ${ctx.method}({ select: { ${
             ctx.firstScalar.name
           }: true } })`
         : ''
 
-      return `Find zero or more ${ctx.plural} that matches the filter.
-${undefinedNote}
-@param {${getModelArgName(ctx.model.name, ctx.action)}=} args - Arguments to filter and select certain fields only.
-@example
-// Get all ${ctx.plural}
-const ${ctx.mapping.plural} = await ${ctx.method}()
-
-// Get first 10 ${ctx.plural}
-const ${ctx.mapping.plural} = await ${ctx.method}({ take: 10 })
-${onlySelect}
-`
+      return dedent`
+        Find one or more '${ctx.singular}' that matches the filter.
```

Comment:
The meaning of the comment was changed. I don't think "one or more" is correct for `findMany`, "zero or more" was correct.